docs/wallet/wallet-extension.md

```diff
@@ -16,112 +14,145 @@ Let's walk through the steps how to install and set up the Maiar DeFi Wallet ext
+    
 
-With this extension installed, you can send to a wallet from your web browser without having to enter your password, which is a key feature of the Maiar DeFi wallet extension.
+### Import existing wallet
 
-![maiar-defi-example-transaction](/wallet/wallet-extension/wallet_extension_step9.png)
+Do you already have a wallet?
 
-Click on "*Send*." Add the wallet address, amount, and data.
+Then there is no need to create a new one. The Maiar Wallet Extension provides an option to import your existing wallet. However, to import an existing wallet you must have access to its **secret (recovery) phrase**.
 
-![maiar-defi-example-send](/wallet/wallet-extension/wallet_extension_step10.png)
+The Maiar wallet has a set of 24-secret phrases, which serve as your wallet’s key. Using your secret phrase to import an existing wallet does not affect your wallet, as all transaction history across all networks will be reflected in the wallet extension.
 
-### Deposit to wallet
+**To get started:** 
 
-To deposit into a wallet, click on "*deposit*" and share your QR code or wallet.
+**Step 1:** With the [Maiar DeFi wallet extension](https://chrome.google.com/webstore/detail/maiar-defi-wallet/dngmlblcodfobpdpecaadgfbcggfjfnm) installed. Click on ‘’Import existing wallet”.
 
-![maiar-defi-example-deposit](/wallet/wallet-extension/wallet_extension_step11.png)
+  ![maiar-defi-wallet-welcome-page](https://user-images.githubusercontent.com/52820835/151935557-3be3f085-40a3-415e-b8a2-afbee54e1313.PNG)
+ 
+**Step 2:** Next, enter your 24-word secret phrase. You can either enter these phrases one at a time or you can simply paste in the phrase using the "_paste_" icon.
 
-### Transactions history
+   ![importwallet](https://user-images.githubusercontent.com/52820835/151940645-b25dcee5-4a37-4950-80e8-434924f9f4d3.PNG)
+  
+**Step 3:** Enter in your wallet password and confirm this password. 
 
-The Maiar DeFi wallet has a record of every transaction that comes in and out of your wallet.
+  ![password3](https://user-images.githubusercontent.com/52820835/151942033-04782d44-982d-44d4-b910-5890a7d256e7.PNG)
+  
+**Step 4:**  Completed! Your Maiar DeFi Wallet has been successfully imported and set to be used.
 
-![maiar-defi-example-transaction](/wallet/wallet-extension/wallet_extension_step12.png)
+   ![successful](https://user-images.githubusercontent.com/52820835/151939047-e8b09e1f-48f7-456b-a6f6-be0a42f83e4e.PNG)
+   
 
-### Networks
+## Key features
 
-In the settings section, there are network options to choose what network you are on. These networks include:
+### Send to a wallet
 
-1. Elrond Testnet
-2. Elrond Devnet
-3. Elrond Mainnet
+Welcome to Maiar DeFi Wallet Extension! One of the key features of this extension is that it allows you to send funds from your wallet to another wallet. To use this feature, you will need to have some funds in your wallet before proceeding. 
```

Comment:
The welcome message should appear earlier, or maybe in a different form. You probably wanted to signal that the "setup" process is now complete and the wallet can now be used, right?

How about something like "Now you have a wallet registered in the Maiar DeFi Wallet Extension and it's ready to use. Great! Here's what you can do with it." Such a phrase could appear immediately after `## Key features`, but before `### Send to wallet` (between the two headings, basically).

Was this the intended effect?